pkg/client/comet_query/client.go

```diff
@@ -0,0 +1,96 @@
+package comet_query
```

Comment:
@bryanchriswhite You asked me to TAL at this in [this comment](https://github.com/pokt-network/poktroll/pull/64#discussion_r1373376300) but it's not something I've ever used or looked into so I honestly don't think there's any value I can provide.

Was there something specific you wanted me to look into or just be aware of this PR?